src/Vehicle/Vehicle.cc

```diff
@@ -3703,6 +3785,17 @@ void Vehicle::triggerCamera()
                    1.0);                            // test shot flag
 }
 
+void Vehicle::operateGripper(bool grip)
+{
+    if(_gripperNumber >= 0) {
+        sendMavCommand(_defaultComponentId,
+                       MAV_CMD_DO_GRIPPER,
+                       true,                  // show errors
+                       (float)_gripperNumber, // gripper servo number
```

Comment:
I don't think `_gripperNumber` is correct. In the above code on Malloy, we have the gripper on Servo7, which would result in `_gripperNumber == 7`, when it should be 1.